python/cudf/cudf/dataframe/index.py

```diff
@@ -32,19 +32,38 @@ def serialize(self):
         transmission.
         """
         header = {}
+        header["index_column"] = {}
+        # store metadata values of index separately
+        # Indexes: Numerical/DateTime/String are often GPU backed
+        if isinstance(self, RangeIndex):
+            header["index_column"]["start"] = self._start
+            header["index_column"]["stop"] = self._stop
+            # We don't need to store the GPU buffer for RangeIndexes
+            frames = []
```

Comment:
Instead of doing this I think it would be better to just implement `serialize` and `deserialize` methods for the RangeIndex object as it would be cleaner.

Also, would you have to add a `frame_count` of 0?

Reply:
Yes!  I should have done this originally